portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro

```diff
@@ -1090,6 +1090,13 @@ definition {
 		WaitForElementNotPresent(locator1 = "ACUtils#LOADING_ANIMATION");
 	}
 
+	macro waitForScrollEvent {
```

Comment:
Hey @calvinctvo, I think the macro could be `waitForScrollDown` since its function to give poshi time can down the page